s3-streamer

```diff
@@ -295,7 +295,9 @@ def main():
         attachments_directory = AttachmentsDirectory(index, tmpdir)
         log_uploader = ChunkedUploader(index, 'log')
 
-        with subprocess.Popen(args.cmd, env=dict(os.environ, TEST_ATTACHMENTS=tmpdir),
+        with subprocess.Popen(args.cmd, env=dict(os.environ,
+                                                 TEST_ATTACHMENTS=tmpdir,
+                                                 LOGDIR_URL=destination.directory),
```

Comment:
Let's call it `TEST_ATTACHMENTS_URL`, and please move this to a separate statement.

Reply:
Done!